docs/Configuration.md

```diff
@@ -0,0 +1,15 @@
+# Configuration
+
+The GUI can be configured by changing the editing the file `assets/app.config.json`.
+
+The supported configuration items are the following.
+
+* `"apiEndpoint"`: The URL to use to contact the API, default `"/api"`.
```

Comment:
Could this either be a full URL, such as "https://zonemaster.fr/api", or the path part, such as "/api"? Please clearify.

Reply:
done

Reply:
It could be either. Without the origin it would use the same origin as the one serving the GUI (as it is with the example apache configuration). We were already using both, https://zonemaster.net/api for the dev / testing environment and `/api` for prod. I tried to clarify a bit the documentation to make that clearer.